ctools/defragment_sharded_collection.py

```diff
@@ -666,9 +666,16 @@ def get_chunk_size_or_0(ch):
 
             # avoid moving larger chunks
             center_size_kb = await get_chunk_size(c)
+
             # Use < 0.6 so that we do not move chunks which were split before
-            if center_size_kb > target_chunk_size_kb * args.small_chunk_frac:
+            has_local_est_size = True if 'defrag_collection_est_size' in c else False
```

Comment:
If we check this after we call `get_chunk_size(c)` than it will be always true. In fact `get_chunk_size(c)` will add the `defrag_collection_est_size` field to the `c` object.

Daniel explained in the email that:
> `chunks_id_index` dictionary is a reference to the original `shard_to_chunks[s]['chunks']` dictionary and changing the `defrag_collection_est_size` in ‘chunks_to_index’ will also change it for `shard_to_chunks`

And I guess that this means that it will also change it for the current processed element `c`

Reply:
I think one solution would be to just move:
```
has_local_est_size = True if 'defrag_collection_est_size' in c else False
```
before line `668`